docs/conf.py

```diff
@@ -0,0 +1,261 @@
+    "https://github.com/collective/collective.easyform#recaptcha-support",
+    "https://github.com/collective/collective.recipe.solrinstance/blob/master/README.rst#multi-core-solr",
+    "https://github.com/nvm-sh/nvm#install-script",
+    "https://github.com/plone/plone.app.contentlisting/#methods-of-contentlistingobjects",
+    "https://github.com/plone/plone.app.contenttypes#changing-the-base-class-for-existing-objects",
+    "https://github.com/plone/plone.recipe.zope2instance#advanced-logging-options-for-wsgi",
+    "https://github.com/repoze/repoze.catalog/blob/master/docs/usage.rst#query-objects",
+    "https://plone.github.io/mockup/dev/#pattern/autotoc",
+    "https://plone.github.io/mockup/dev/#pattern/modal",
+    "https://plone.github.io/mockup/dev/#pattern/moment",
+    "https://github.com/collective/collective.exportimport#faq-tips-and-tricks",
+    "https://github.com/plone/plone.app.contenttypes/tree/2.2.x#migration",
+    "https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Keywords",
+    "https://solr.apache.org/guide/8_2/updatehandlers-in-solrconfig.html#UpdateHandlersinSolrConfig-commitWithin",
+    "https://www.npmjs.com/package/axios#example",
+    "https://github.com/plone/plone.restapi/blob/afde2a940d2518e061eb3fe30093093af55e3a50/src/plone/restapi/services/content/configure.zcml#L15-L20",
+    "https://github.com/plone/plone.rest#cors",
+    "https://github.com/plone/plone.docker#for-basic-usage",
+    "https://github.com/nodejs/release#release-schedule",
+    # ### End of list of anchored links
+]
+linkcheck_allowed_redirects = {
+    # All HTTP redirections from the source URI to the canonical URI will be treated as "working".
+    r"https://chrome\.google\.com/webstore/detail/.*": r"https://consent\.google\.com/.*",
+}
+linkcheck_anchors = True
+linkcheck_timeout = 10
+linkcheck_retries = 2
+
+# This is our wordlist with known words, like Github or Plone ...
+spelling_word_list_filename = "spelling_wordlist.txt"
+spelling_ignore_pypi_package_names = True
+
+# The suffix of source filenames.
+source_suffix = {
+    ".md": "markdown",
+}
+
+# The master toctree document.
+master_doc = "index"
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = [
+    "spelling_wordlist.txt",
+]
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "sphinx_book_theme"
+html_logo = "_static/logo.svg"
+html_favicon = "_static/favicon.ico"
+
+html_css_files = ["custom.css", ("print.css", {"media": "print"})]
+
+html_extra_path = [
+    "robots.txt",
+]
+
+# Used by sphinx_sitemap to generate a sitemap
+html_baseurl = "https://training.plone.org"
```

Comment:
This needs to be wherever you host it, else the default would be `https://6.docs.plone.org/`.